invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html

```diff
@@ -60,16 +60,28 @@
             {%- if config.THEME_SEARCHBAR %}
               {%- block navbar_search %}
                 <div class="item p-0 search-bar rel-mr-1">
-                  <div role="menuitem" id="header-search-bar">
-                    <form class="form" action="{{ config.THEME_SEARCH_ENDPOINT }}"
-                          role="search">
-                      <div class="ui action icon input">
-                        <input type="text" name="q" placeholder="{{ _('Search') }}">
-                        <button type="submit" class="ui icon search button" aria-label="{{ _('Search') }}">
-                          <i class="icon search"></i>
-                        </button>
+                  {% set options = 
+                    [
+                      {
+                        "key": "records",
+                        "text": _("All") + " " + config.THEME_SITENAME,
```

Comment:
```suggestion
                        "text": _("Everywhere"),
```

Reply:
This was discussed in Discord, ping @jennur and @kpsherva to re-think this.